docs/external-etcd.md

```diff
@@ -0,0 +1,67 @@
+# Support for external etcd
```

Comment:
We should embed this an advanced section in the CAPI book

Reply:
:D yes!